base/status/StatusStatistics.ts

```diff
@@ -0,0 +1,376 @@
+        let cursor_x = 0;
+        let cursor_y = 0;
+
+        if(this.current_col === 0){
+            highlight.x = StatusStatistics.HIGHLIGHT[this.current_line].X;
+            highlight.y = StatusStatistics.HIGHLIGHT[this.current_line].Y;
+            highlight.width = StatusStatistics.HIGHLIGHT[this.current_line].WIDTH;
+            highlight.height = StatusStatistics.HIGHLIGHT[this.current_line].HEIGHT;
+
+            cursor_x = StatusStatistics.CURSOR[this.current_line].X;
+            cursor_y = StatusStatistics.CURSOR[this.current_line].Y;
+        }
+        else{
+            if(this.manager.battle_effects_array.length === 0){
+                highlight.x = StatusStatistics.HIGHLIGHT.NORMAL.X;
+                highlight.y = StatusStatistics.HIGHLIGHT.NORMAL.Y;
+                highlight.width = StatusStatistics.HIGHLIGHT.NORMAL.WIDTH;
+                highlight.height = StatusStatistics.HIGHLIGHT.NORMAL.HEIGHT;
+            }
+            else{
+                let highlight_shift = StatusStatistics.HIGHLIGHT.EFFECT.SHIFT;
+                highlight.x = StatusStatistics.HIGHLIGHT.EFFECT.X + highlight_shift*(this.current_col-1);
+                highlight.y = StatusStatistics.HIGHLIGHT.EFFECT.Y;
+                highlight.width = StatusStatistics.HIGHLIGHT.EFFECT.WIDTH;
+                highlight.height = StatusStatistics.HIGHLIGHT.EFFECT.HEIGHT;
+            }
+            
+            let cursor_shift = StatusStatistics.CURSOR.EFFECT.SHIFT;
+            cursor_x = StatusStatistics.CURSOR.EFFECT.X + cursor_shift*(this.current_col-1);
+            cursor_y = StatusStatistics.CURSOR.EFFECT.Y;
+        }
+        this.update_highlight(highlight);
+        
+        let cursor_tween = {type: CursorManager.CursorTweens.POINT, variant: PointVariants.SHORT};
+        this.data.cursor_manager.move_to({x: cursor_x, y:cursor_y}, {animate: false, flip: true, tween_config: cursor_tween});
+    }
+
+    public on_change(){
+        this.select_option();
+        if(this.current_col===0){
+            let msgs = {line1: StatusStatistics.StatisticsMsgs[this.current_line].line1,
+                line2: StatusStatistics.StatisticsMsgs[this.current_line].line2};
+
+            if(this.current_line === Statistics.EXP){
+                let char = this.manager.selected_character;
+                let exp = char.exp_curve[char.level] - char.current_exp;
+                msgs.line2 = msgs.line2.replace("${EXP}", exp);
+            }
+            this.manager.update_description(msgs.line1, msgs.line2);
+        }
+        else{
+            if(this.manager.battle_effects_array.length === 0){
+                this.manager.update_description("Normal status.", "");
+            }
+            else{
+                let effect = this.manager.battle_effects_array[this.current_col-1];
+
+                let msgs = null;
+                if(effect.type === BattleEffectTypes.STATUS_CONDITION){
+                    let name = (effect.key as string).toUpperCase();
```

Comment:
You could change `BattleStatusEffect.key` type maybe to `temporary_status|permanent_status`